### ISSUE TITLE
How to disable Beacon logs to console?

### ISSUE THREAD

**Issue by apetta** [ecadlabs/taquito#2613]
Hi,

Sorry if I've labelled this issue wrong, but I can't see any docs on how to prevent logging to the browser console when interacting with @taquito/beacon-wallet.

How can I turn that off? Is it only a dev environment thing?

**Comment by dsawali** [comment]
Hey there @apetta. Could you elaborate what logs you mean?

A screenshot might help maybe

**Comment by apetta** [comment]
Of course. It starts when you attempt to connect, they're quite overwhelming. I did see a `setDebugEnabled()` option in the [beacon-sdk changelog](https://docs.walletbeacon.io/CHANGELOG), but I have no idea if that has anything to do with these logs, and if so - how to set it when using beacon via taquito.

![image](https://github.com/ecadlabs/taquito/assets/33671231/1558c8f5-a2a9-46a5-a89a-199d8e7d5c8f)

**Comment by dsawali** [comment]
Ah I understand now. These aren't really in our control unfortunately. You could probably reach out to their repo and create an issue there @apetta

**Comment by Innkst** [comment]
We see the dependent Beacon SDK issue is merged.
Next time we will update our Beacon dependency, logging updates should work.

**Comment by AlirezaHaghshenas** [comment]
Related: https://github.com/airgap-it/beacon-sdk/issues/656

**Comment by Innkst** [comment]
We have update beacon-sdk part of v17.5.1, so this should not be an issue any longer.
